pinterest/gestalt
# Docs: Move props back to the top + make collapsible

1/3rd of the survey responses mentioned they wanted to see `Prop` back on the top of the docs. In order to also make this work for designers, we added a collapsible feature:

![props-demo-4](https://user-images.githubusercontent.com/127199/98152553-c5ec9800-1e86-11eb-84e6-68924596606a.gif)

The state whether to collapse props is saved in local storage so it's maintained across page navigation and refreshes.

## Test Plan

1. Go to `/Icon`
2. Click the toggle button to collapse the props
3. Props should collapse
